src/style.css

```diff
@@ -1,3 +1,60 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  text-transform: uppercase;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: black;
+  height: 60px;
+  display: block;
+  line-height: 60px;
+  text-decoration: none;
+}
+
+.nav__link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  color: rgba(0, 172, 220, 1);
+  border-bottom: 4px solid rgba(0, 172, 220, 1);
```

Comment:
Don't use a border for this purpose. Use `::after` and add a rounded border as in the mock